configs/config.properties

```diff
@@ -1,6 +1,6 @@
 #APP_PATH=/Users/saikrisv/Downloads/***.ipa
 RUNNER=distribute
-FRAMEWORK=testng
+FRAMEWORK=cucumber
```

Comment:
Please revert it as it might affect our CI runs. Looks like this is an accidental check-in.